site/pages/api/feedback.ts

```diff
@@ -0,0 +1,132 @@
+    const refinedGeneralFeedbackInput = removeExcessWhiteSpace(body.generalFeedbackQuestion);
+    if (refinedHearAboutServiceInput && refinedHearAboutServiceInput !== "") {
+        feedback.push({
+            question: hearAboutUsFeedbackQuestion,
+            answer: refinedHearAboutServiceInput,
+        });
+    }
+    if (refinedGeneralFeedbackInput && refinedGeneralFeedbackInput !== "") {
+        feedback.push({
+            question: generalFeedbackQuestion,
+            answer: refinedGeneralFeedbackInput,
+        });
+    }
+    if (body.contactQuestion) {
+        feedback.push({
+            question: contactFeedbackQuestion,
+            answer: body.contactQuestion,
+        });
+    }
+    if (body.problemQuestion) {
+        feedback.push({
+            question: solveFeedbackQuestion,
+            answer: body.problemQuestion,
+        });
+    }
+
+    return feedback;
+};
+
+export const requestIsEmpty = (req: FeedbackApiRequest): boolean => {
+    const { body } = req;
+    const refinedHearAboutServiceInput = removeExcessWhiteSpace(body.hearAboutServiceQuestion);
+    const refinedGeneralFeedbackInput = removeExcessWhiteSpace(body.generalFeedbackQuestion);
+    if (
+        !body.contactQuestion &&
+        !body.problemQuestion &&
+        refinedHearAboutServiceInput === "" &&
+        refinedGeneralFeedbackInput === ""
+    ) {
+        return true;
+    }
+
+    return false;
+};
+
+// redactEmailAddress replaces the user portion of an email address so it can be safely logged
+export const redactEmailAddress = (
+    toRedact: string | Mail.Address | (string | Mail.Address)[] | undefined,
+): string | string[] => {
+    const redact = (address: string): string => address.toString().replace(/.*@/, "*****@");
+    if (toRedact !== undefined) {
+        if (typeof toRedact === "string") {
+            return redact(toRedact);
+        } else if (toRedact.hasOwnProperty("name") && toRedact.hasOwnProperty("address")) {
+            const email = toRedact as Mail.Address;
+            return redact(email.address);
+        } else if (typeof toRedact === "object") {
+            const addresses = toRedact as Mail.Address[];
+            return addresses.map((email) => {
+                if (email.hasOwnProperty("name") && email.hasOwnProperty("address")) {
+                    return redact(email.address);
+                } else {
+                    return redact(email as unknown as string);
+                }
+            });
+        }
+    }
+    return "*****@*****.***";
+};
+
+const feedback = (req: FeedbackApiRequest, res: NextApiResponse): void => {
+    let mailOptions: Mail.Options = {};
+    try {
+        if (requestIsEmpty(req)) {
+            redirectTo(res, "/feedback?feedbackSubmitted=false");
+            return;
+        }
+
+        const feedback: Feedback[] = buildFeedbackForEmail(req);
+        const feedbackSubmitterEmailAddress = getEmailFromIdToken(req) || "UNKNOWN";
```

Comment:
Just hardcode this to UNKNOWN for the time being

Reply:
done